gaia-media-controls.js

```diff
@@ -0,0 +1,291 @@
+    display: flex;
+    flex-flow: row;
+    align-items: center;
+    font-size: 0;
+    border-bottom: 0.1rem solid rgba(255,255,255, 0.1);
+  }
+
+  /* 1. elapsed-text and duration-text have padding on left and right
+        to support ltr and rtl locales */
+  /* 2. The elapsed time and duration elements do not grow and shrink
+        via the flexbox. They are fixed width */
+  .elapsed-text, .duration-text {
+    color: #ffffff;
+    font-size: 1.4rem;
+    padding: 0 1.5rem; /* 1 */
+    flex-grow: 0;      /* 2 */
+    text-align: center;
+  }
+
+  /* 1. The slider element grows and shrinks via the flexbox */
+  .slider-wrapper {
+    flex-grow: 1;   /* 1 */
+    height: 4.2rem;
+  }
+
+  .progress {
+    position: relative;
+    pointer-events: none;
+    width: 0;
+  }
+
+  /* 1. Center elements vertically within time-slider; 'top: 50%' centers the
+   *    top of the element vertically; 'elapsed-time' is the first element
+   *    to be layed out, it is 0.3rem in height, therefore 'top:50%' would
+   *    position the center of the element 0.1rem below the middle: move
+   *    it up 0.1rem to center the middle of the element vertically.
+   *    'time-background' is layed out after 'elapsed-time' and is 0.1rem in
+   *    height. With 'elapsed-time' being 0.3rem in height, 'top:50%' would
+   *    position 'time-background' 0.3rem below the center vertically: move it
+   *    up 0.3rem to center it vertically.
+   *
+   * 2. Ensure the layering order of time background,
+        elapsed time, and play head.
+   */
+
+  .elapsed-time {
+    height: 0.3rem;
+    background-color: #00caf2;
+    top: calc(50% - 0.1rem); /* 1 */
+    z-index: 20; /* 2 */
+  }
+
+  .time-background {
+    width: 100%;
+    height: 0.1rem;
+    top: calc(50% - 0.3rem); /* 1 */
+    background-color: #a6b4b6;
+    z-index: 10; /* 2 */
+  }
+  /*
+   * 1. Center 'play-head' vertically. 'top' is relative to the other
+   *    'slider-wrapper' elements which are 0.4rem in height; therefore,
+   *    'top:50%' would position the top of the element 0.4rem below the
+   *    the center of the 'sider-wrapper'. In order to position the center
+   *    of the element vertically, move it up by half its height plus
+   *    the height of the previously layed out elements.
+   *
+   * 2. Ensure the layering order of time background,
+   *    elapsed time, and play head.
+   */
+  .play-head {
+    top: calc(50% - (1.15rem + 0.4rem));
+    position: relative;
+    width: 2.3rem;
+    height: 2.3rem;
+
+    /* For LTR langauges, position the playhead 1.15 rems to the left
+     * so that the center of the playhead aligns with the beginning of
+     * the time slider.
+     */
+    margin-left: -1.15rem;
+
+    /* For RTL langauges, position the playhead 1.15 rems to the right
+     * so that the center of the playhead aligns with the end of
+     * the time slider.
+     */
+    margin-right: -1.15rem;
+
+    border: none;
+    background: none;
+    pointer-events: none;
```

Comment:
I'm surprised by pointer-events: none. Aren't you handling drags on this element?  Or is it just drags anywhere on the time line?